platform/ui/src/components/AllInOneMenu/Item.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { ReactNode, useCallback, useContext } from 'react';
+import { MenuContext } from './Menu';
+
+type ItemProps = {
+  label: string;
+  secondaryLabel?: string;
+  icon?: ReactNode;
+  onClick?: () => void;
+};
+
+const Item = ({ label, secondaryLabel, icon, onClick }: ItemProps) => {
+  const { hideMenu } = useContext(MenuContext);
+
+  const onClickHandler = useCallback(() => {
+    hideMenu();
+    onClick?.();
+  }, [hideMenu, onClick]);
+
+  return (
+    <div
+      className="all-in-one-menu-item all-in-one-menu-item-effects"
+      onClick={onClickHandler}
+    >
+      {icon && (
+        <>
+          {icon}
+          <div className="w-2"></div>
+        </>
+      )}
+      <span>{label}</span>
+      {secondaryLabel != null && (
+        <span className="text-aqua-pale ml-[1ch]">{`${secondaryLabel}`}</span>
```

Comment:
what is `1ch` never saw that in tailwind

Reply:
I guess you can just say {secondaryLabel}

Reply:
`1ch` is not from tailwind but a pure css unit https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units. It is used here to get a one character separation between the primary and secondary labels.